types.go

```diff
@@ -16,3 +16,8 @@ var (
 	ErrCacheNotSupported = errors.New("Operation not supported")
 )
 
+//Interface for all onecache store implementations
+type CacheStore interface {
+	Set(key string, data interface{}, expires time.Duration) error
+	Get(key string) ([]byte, error)
```

Comment:
I decided to go with returning the data as is (i.e an `interface{}`).. Allows for much more usability.

https://github.com/adelowo/onecache/pull/1/files#diff-e871404fd69cab40521d6b2178847cf7R28